src/server/services/hli/stadspas-gpass-service.ts

```diff
@@ -250,33 +250,31 @@ export async function fetchGpassBudgetTransactions(
 
 function transformGpassAanbiedingenResponse(
   responseSource: StadspasDiscountTransactionsResponseSource
-): StadspasDiscountTransactions {
-  const discountAmountTotal = responseSource.totale_korting ?? 0;
-  return {
-    discountAmountTotal,
-    discountAmountTotalFormatted: `€${displayAmount(Math.abs(discountAmountTotal))}`,
-    transactions: parseTransactions(responseSource.transacties),
-  };
+): StadspasDiscountTransactions | StadspasDiscountTransactionsResponseSource {
+  if (Array.isArray(responseSource?.transacties)) {
+    const discountAmountTotal = responseSource.totale_korting ?? 0;
+    return {
+      discountAmountTotal,
+      discountAmountTotalFormatted: `€${displayAmount(Math.abs(discountAmountTotal))}`,
+      transactions: transformTransactions(responseSource.transacties),
+    };
+  }
+  return responseSource;
```

Comment:
Mja, ik zei net wat anders maar misschien moeten we hier toch maar een default type meegeven. Dus een `[]` in dit geval. Wat vind jij?